oozie-to-airflow/converter/parsed_node.py

```diff
@@ -22,17 +22,25 @@
 # Pylint and flake8 does not understand forward references
 # https://www.python.org/dev/peps/pep-0484/#forward-references
 from mappers import base_mapper  # noqa: F401 pylint: disable=unused-import
+from converter import primitives  # noqa: F401 pylint: disable=unused-import
 
 
 class ParsedNode:
     """Class for parsed Oozie workflow node"""
 
-    def __init__(self, mapper: "base_mapper.BaseMapper"):
+    def __init__(
+        self,
+        mapper: "base_mapper.BaseMapper",
```

Comment:
We should use classes here not strings. That makes it easier for IntelliJ usage and it makes use of import statements which is good to show that there are no hidden dependencies.

In case it causes a circular dependency, then it should be fixed in the structure or (worst case) we can use local imports in this case.